microsoft/vscode
# add ignoreSubmodules option

This PR fixes https://github.com/microsoft/vscode/issues/88715
_(now that it has ~~for when it reaches~~ 20 up-votes)_

**Purpose:** Adds a setting to ignore modifications to submodules in the file tree so that large monorepos don't show numerous modifications.

**Testing:**
- Open VSCode to a repo with submodules
- Go to settings and find the "Ignore Submodules" setting
- Check and uncheck the box to watch the 'M' convert to an 'S'
